zanata-war/src/main/java/org/zanata/action/MergeTransAction.java

```diff
@@ -184,6 +186,9 @@ public void startMergeTranslations() {
         mergeTranslationsManager.start(sourceProjectSlug,
             sourceVersionSlug, targetProjectSlug, targetVersionSlug,
             !keepExistingTranslation);
+        FacesContext context = FacesContext.getCurrentInstance();
+        NavigationHandler navigationHandler = context.getApplication().getNavigationHandler();
+        navigationHandler.handleNavigation(context, null, "merge-translation");
```

Comment:
Why do we need this? Shouldn't this be going to the same page where the action is triggered by default?
Also, sounds like it's something that would commonly be used so maybe we should put it in some common class.